### PR TITLE
styled posts and removed unnecessary files

### PR DESCRIPTION
**Pull request by bmbaron**

### CODE REVIEW

**Comment by bmbaron** [comment]
image

Laptop

image

Mobile

File: public/index.html [hunk 1]
```diff
@@ -2,28 +2,11 @@
...
-    <link rel="manifest" href="%PUBLIC_URL%/manifest.json" />
-    <!--
-      Notice the use of %PUBLIC_URL% in the tags above.
-      It will be replaced with the URL of the `public` folder during the build.
-      Only files inside the `public` folder can be referenced from the HTML.
-
-      Unlike "/favicon.ico" or "favicon.ico", "%PUBLIC_URL%/favicon.ico" will
-      work correctly both with client-side routing and a non-root public URL.
-      Learn how to configure a non-root public URL by running `npm run build`.
-    -->
```
Comment (qkudev) [inline public/index.html]: Why did you deleted these lines?

Actually, I don't think you should do it. But really, what is the reason?

File: src/App.tsx [hunk 2]
```diff
@@ -12,10 +11,11 @@ function App() {
     setData(rest);
   }
   return (
-    <div>
-      <button onClick={removeSecondPost}>remove 2nd post</button>
+    <MainContainer>
+      {/*<button onClick={removeSecondPost}>remove 2nd post</button>*/}
```
Comment (qkudev) [inline src/App.tsx]: Remove this comment, please

I does nothing

File: src/app.styled.ts [hunk 3]
```diff
@@ -0,0 +1,18 @@
+import styled from "styled-components";
+
+export const MainContainer = styled('div')`
```
Comment (qkudev) [inline src/app.styled.ts]: Use `main`, please

It is a good practice for Search Engines and screen readers to use semantic HTML tags.

Take a look [here](https://developer.mozilla.org/en-US/docs/Glossary/Semantics#semantics_in_html)

And [there is](https://web.dev/learn/accessibility/) also a good accessibility course by Google developers

File: src/components/PostsList/postsLists.styled.ts [hunk 4]
```diff
@@ -1,7 +1,33 @@
 import styled from "styled-components";
 
 export const List = styled('ul')`
+  width: auto;
+  margin: 20px auto;
```
Comment (qkudev) [inline src/components/PostsList/postsLists.styled.ts]: I see you are using much hard coded spacings like margin and padding

It is a good practice to use `theme` from `styled-components`

Could you please add `xstyled`? Lets start with spacings and typography!

You can check out the docs [here](https://xstyled.dev/docs/magic-styled-components/)

File: src/components/Post/Post.tsx [hunk 5]
```diff
@@ -1,15 +1,16 @@
 import React, {memo} from 'react';
-import {PostContainer, Title, Description, DateContainer} from "./post.styled";
+import {PostContainer, Title, Description, DateContainer, Button} from "./post.styled";
 type Props = IPost
 export const Post = memo(({id, title, description, createdAt}: Props) => {
   console.log("id: ", id);
   return (
       <PostContainer>
         <Title>{title}</Title>
```
Comment (qkudev) [inline src/components/Post/Post.tsx]: Hey!

I think you could add `ui` directory with components that do not use any business logic and have only stylings and presentations.

Could you please create `ui/Title` directory and add common `Title` component?

**Review by qkudev — CHANGES_REQUESTED** [review]

**Comment by qkudev** [comment]
Also it is a good practice to add description to your pull request with content like:

```
// markdown
# Changes

- adds `Post` and `PostList` components
- adds mocked posts data
```

**Comment by qkudev** [comment]
Let me know when you add `xstyled` and then we will continue with `react-router` and app pages!

File: public/index.html (hunk 1 above)
Comment (bmbaron) [reply public/index.html]: Adding back in. I think I was a bit too "trigger happy" when trying to remove the boilerplate code.

**Pull request closed without merging**